drools-parser/src/main/antlr4/org/drools/parser/DRLLexer.g4

```diff
@@ -18,16 +18,19 @@ import JavaLexer;
 // KEYWORDS
 /////////////////
 
-DRL_PACKAGE : 'package';
+// These keywords duplicate ones in JavaLexer. They should not be overriden with different names, or else Vocabulary's literalName will be null.
```

Comment:
Hi @tkobayas 
I can not follow: if those are "duplicates", why we have to declare again ? Just to understand....

Reply:
@gitgabrio Thank you, you are right. I had a screw loose while debugging back and forth :D
No need to declare them again. I keep them as comments for now because they were enlisted as `DroolsSoftKeywords` in the current (antlr3) implementation.